### ISSUE TITLE
Anyway to implement a trash can?

### ISSUE THREAD

**Issue by bluelavalamp** [trapexit/mergerfs#714]
Is there any way using mergerfs that I could implement a trash can?  So when a file is deleted instead of actually deleting the file it would be moved to a different folder (the trash can).  

Thanks

**Comment by trapexit** [comment]
Yes. It could use the freedesktop.org standard. The main problem I'd see is that people tend not to expect that behavior outside GUI apps. You'd need to ensure you have proper tooling installed so you can manage the trashcan otherwise you'd fill your drives unknowingly. I wouldn't want to add some sort of trash pruning or whatnot.
